### PR TITLE
completed get/users returning array of user objects #21

### PR DESCRIPTION
**Pull request by kam-cheng**
completed get/users returning array of user objects #21

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -25,3 +25,19 @@ describe("GET/api/topics", () => {
...
+  test("200 - returns 200 Status", async () => {
+    await request(app).get("/api/users").expect(200);
+  });
+  test("200 - returns array of objects containing username property", async () => {
+    const users = await request(app).get("/api/users").expect(200);
+    users.body.forEach((slug) => {
+      expect(slug).toEqual(
+        expect.objectContaining({
+          username: expect.any(String),
+        })
```
Comment (daviidbartlett) [inline __tests__/app.test.js:39]: can we assert we can the correct amount back please?

Comment (DCrawley94) [reply __tests__/app.test.js:43]: Looks good! I'd advise combining the length test with your assertions against the content to tidy things up.

File: controllers/app.controllers.js [hunk 2]
```diff
@@ -7,5 +7,5 @@ exports.getTopics = async (req, res) => {
 
 exports.getUsers = async (req, res) => {
   const users = await fetchUsers();
-  res.status(200).send(users);
+  res.status(200).send({ users });
```
Comment (DCrawley94) [inline controllers/app.controllers.js:10]: Perfect!

**Review by DCrawley94 — COMMENTED** [review]
Looking good!

Cheers for making the changes

**Pull request merged**